Refer to . 

[Leveraging big data to transform target selection and drug discovery](https://www.ncbi.nlm.nih.gov/pubmed/26659699).  
[Harnessing big ‘omics’ data and AI for drug discovery in hepatocellular carcinoma](https://www.nature.com/articles/s41575-019-0240-9).  

Data Types include Genomics (G), Epigenomics (E), Transcriptomics (T), Proteomics (P), Metabolomics (M), Phenomics (P), Imaging (I) .   
Mainly list the big data repositories

# Disease models
## Clinical models 

## Preclinical models

# Perturbagen 

## OMICS
(***) [A Library of Phosphoproteomic and Chromatin Signatures for Characterizing Cellular Responses to Drug Perturbations](https://www.biorxiv.org/content/early/2017/09/07/185918).  
profiled 90 drugs (in triplicate) in six cell lines using two different proteomic assays

(*) [Orthotopic patient-derived xenografts of paediatric solid tumours](https://www.nature.com/nature/journal/v549/n7670/full/nature23647.html).  
sensitivity data of 150 drugs across >20 pediatric cell lines

(*) [NCI ALMANAC](https://dtp.cancer.gov/ncialmanac/initializePage.do)
5,000 pairs of FDA-approved drugs that were tested against the NCI-60

# Single cells
(***) Liver cancer single cells ([Landscape of Infiltrating T Cells in Liver Cancer Revealed by Single-Cell Sequencing](http://www.cell.com/cell/fulltext/S0092-8674(17)30596-2)).  
5,063 single T cells isolated from peripheral blood, tumor, and adjacent normal tissues from six hepatocellular carcinoma patients.  
